tests/testthat/test-bayes.R

```diff
@@ -447,6 +447,19 @@ test_that("too few starting values", {
   expect_snapshot(error = TRUE, tune:::check_bayes_initial_size(1, 1, FALSE))
 })
 
+test_that("error if wrong control function is used", {
+  set.seed(4400)
+  wflow <- workflow() %>% add_recipe(rec_tune_1) %>% add_model(lm_mod)
+  pset <- dials::parameters(wflow) %>% update(num_comp = dials::num_comp(c(1, 5)))
+  folds <- vfold_cv(mtcars)
```

Comment:
Could you namespace calls to rsample/dials/parsnip in these tests?

Reply:
done